src/components/collapsible-section/collapsible-section.scss

```diff
@@ -65,7 +65,8 @@
 
 .section__body {
     background-color: var(--body-background-color, #ffffff);
-    padding: var(--body-padding, 0);
+    padding: var(--body-padding, pxToRem(20));
```

Comment:
I don't think this is a breaking change. But should we somehow mention it that this affects spacing of the contents inside sections so that developers know about it? (it makes it better, but you never know what people have put in their sections.)